indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -56,7 +56,10 @@
                             {% for item in dynamic_columns %}
                                 <th class="i-table">{{ item.title }}</th>
                             {% endfor %}
-                            <th class="i-table thin-column"></th>
+                            {% if reviewed_abstracts %}
```

Comment:
I wouldn't gray out the icon, as if you haven't reviewed anything it may look like all of them are reviewed. OTOH, I would not hide the column.